packages/fx-core/templates/core/v3Migration/upgradeReport.md

```diff
@@ -37,6 +37,8 @@ For more detailed changes, please refer to this [wiki](https://aka.ms/teams-tool
 
 2. If your project is created with Viusal Studio version < 17.4, you may see error `InvalidParameter: Following parameter is missing or invalid for aadApp/create action: name` when executing commands. Please follow [the steps](#how-to-roll-back) to roll back, install VS 17.4 and run upgrade first.
```

Comment:
Not blocking this PR. @KennethBWSong is this known issue still valid?

Reply:
I think it is still valid as long as we don't have any update on migration of projects before 17.4.